brainscore_core/plugin_management/test_plugin.sh

```diff
@@ -7,6 +7,8 @@ PLUGIN_TEST_PATH=$PLUGIN_PATH/test.py
 SINGLE_TEST=$3
 CONDA_ENV_PATH=$PLUGIN_PATH/environment.yml
 LIBRARY_PATH=$4
+PYTHON_VERSION=$(python -V | cut -d ' ' -f 2 | cut -d '.' -f 1,2)
```

Comment:
```suggestion
PYTHON_VERSION=$(python -c "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}.{sys.version_info.micro}')")
```
Maybe more stable / readable

Reply:
Yup, definitely more readable.